### ISSUE TITLE
Compilation error in libevent.nit

### ISSUE THREAD

**Issue by matthmsl** [nitlang/nit#2684]
## What happend ?
Tried to test modifications on libevent.nit with ```nitc libevent.nit```
## Expected result
None. Should have compiled silently
## Output
```bash
~/Bureau/nit/lib/libevent$ nitc libevent.nit 
libevent.nit:244,12--35: Error: no property `Connection::write_bytes_from_cstring` is inherited. Remove the `redef` keyword to define a new property.
		redef fun write_bytes_from_cstring(ns, len)
		                         ^
```
Then 
```bash
~/Bureau/nit/lib/libevent$ nitc libevent.nit 
libevent.nit:244,31--32: Error: untyped parameter `ns'.
		fun write_bytes_from_cstring(ns, len)
		                             ^
Errors: 1. Warnings: 0.
```
Then
```bash
~/Bureau/nit/lib/libevent$ nitc libevent.nit 
libevent.nit:244,45--47: Error: untyped parameter `len'.
		fun write_bytes_from_cstring(ns : CString, len)
		                                           ^
Errors: 1. Warnings: 0.
```
## Note
Modifications done to libevent couldn't have created this issue. It seems to work when I replace 
```nit
redef fun write_bytes_from_cstring(ns, len)
	do
		if close_requested then return
		native_buffer_event.write(ns, len)
	end
```
with
```nit
	fun write_bytes_from_cstring(ns: CString, len: Int)
	do
		if close_requested then return
		native_buffer_event.write(ns, len)
	end
```

**Comment by matthmsl** [comment]
My nitc has been compiled this morning... As for libevent it hasn't been modified for 13 days and it was just a change of location.

**Comment by R4PaSs** [comment]
Strange, could you try to rebase your changes on the current master branch? The `write_bytes_from_cstring` service was introduced with #2642, merged 10 days ago, if the commit d67eec0 is in your tree, this should not be an issue.
On the current master, I've tried running `nitc lib/libevent/libevent.nit`, this did not produce any error.

**Comment by matthmsl** [comment]
Well, the rebase work in hard mode... Something may have not been updated with the nice soft way. Strange but okay I guess... 
I'll just assume it was just the ground.
